resolwe/flow/executors/docker/__init__.py

```diff
@@ -30,15 +34,45 @@ def start(self):
             'container_image': self.requirements.get('image', settings.FLOW_EXECUTOR['CONTAINER_IMAGE']),
         }
 
+        # Set resource limits.
+        limits = []
+        if 'cores' in self.resources:
+            # Each core is equivalent to 1024 CPU shares. The default for Docker containers
+            # is 1024 shares.
+            limits.append('--cpu-shares={}'.format(int(self.resources['cores']) * 1024))
+
+        if 'memory' in self.resources:
```

Comment:
@jkokosar, @mstajdohar do you have any suggestion? Maybe 1 core and 4-8gb of memory?